babel.config.js

```diff
@@ -53,8 +53,7 @@ module.exports = (api) => {
     ...(DEV || TEST
       ? {
           alias: {
-            '^@apache-annotator/([^/]+)$': ([, name]) =>
-              path.join(packagePath, name, '/src/index.ts'),
+            '^@apache-annotator/([^/]+)$': `${packagePath}/\\1/src/index.ts`,
```

Comment:
This last commit is exactly reversing the fix for Windows in https://github.com/apache/incubator-annotator/pull/105/files

Reply:
Good catch. Pushed up another commit to fix that.